### PR TITLE
Use latest version of NPM and Node.js

### LINKED ISSUE #4558: Use latest version of NPM and Node.js
**Is your feature request related to a problem? Please describe.**
Use the latest version of NPM and Node.js.

**Describe the solution you'd like**
Bump up to the latest version of NPM and Node.js.

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by FK78**
Partially resolves [#743](https://github.com/bbc/simorgh-infrastructure/issues/743)

**Overall change:** _Bump `.npmrc` and `.nvmrc`._

**Code changes:**

- _Change NPM version from 6.9.0 to 6.12.0._
- _Change Node.js version from 10.16.3 to 12.13.0._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by DemonusPC — APPROVED** [review]
Dank

File: package-lock.json [hunk 1]
```diff
@@ -1413,7 +1413,7 @@
       "integrity": "sha512-L92D9HyBqMORYy6u3wvkGq4CO1SVQ+SYub1fbxH/IXteSemr3pVpJrSGy3NIsKQcsAyqMcYfsacE/MMEcmdldw==",
       "requires": {
         "@bbc/psammead-image": "^1.2.3",
-        "@bbc/psammead-play-button": "1.0.0"
+        "@bbc/psammead-play-button": "^1.0.0-alpha.0"
```
Comment (chris-hinds) [inline package-lock.json]: Is this expected?

File: package-lock.json [hunk 2]
```diff
@@ -1441,8 +1441,15 @@
       "integrity": "sha512-xoBr+l/fijNKE0cZK4TFsbyJmwq2N9o1zy/qOqgx+V+KR59HUbPfmEaBPzoFbgufYSC5mHWeyYf6IHezOoNh8w==",
       "requires": {
         "@bbc/gel-foundations": "^3.4.1",
-        "@bbc/psammead-assets": "^2.6.0",
+        "@bbc/psammead-assets": "^2.7.0",
```
Comment (chris-hinds) [inline package-lock.json]: Is this expected?

File: package-lock.json (hunk 1 above)
Comment (FK78) [reply package-lock.json]: Reset.

File: package-lock.json (hunk 2 above)
Comment (FK78) [reply package-lock.json]: Reset.

**Review by DemonusPC — APPROVED** [review]
Dank

**Review by chris-hinds — APPROVED** [review]

**Review by DemonusPC — APPROVED** [review]

**Review by sareh — APPROVED** [review]

**Pull request merged**